internal/sirius/edit_case_test.go

```diff
@@ -34,6 +34,7 @@ func TestEditCase(t *testing.T) {
 						Path:   dsl.String("/lpa-api/v1/lpas/800"),
 						Body: map[string]interface{}{
 							"status": "Suspended",
+							"donor":  Person{ID: 1},
```

Comment:
I don't think this should be here: you can't change the donor when editing an LPA, it'll just be ignored by the API

Reply:
It doesn't seem to be ignored, at least when i run the tests locally

`Error: Expected nil, but got: &errors.errorString{s:"Actual interactions do not match expected interactions for mock MockService.\n\nIncorrect requests:\n\tPUT /lpa-api/v1/lpas/800 (request body did not match)\n\nDiff with interaction: \"A request to edit the LPA\" given \"I have a pending case assigned\"... Could not find key \"donor\" (keys present are: status) at $.body\n\nSee /Users/hibo.abdilaahi/Documents/GitHub/opg-sirius-lpa-frontend/logs/pact.log for details.\n"}`

The solution seems to be to cast the donor field to a pointer type `*Person`